region/maridia/inner-pink.json

```diff
@@ -1440,6 +1440,78 @@
                     "Even when suitless, it's possible to stand far enough that the acid attack doesn't spawn.",
                     "Botwoon still gets hit. So there's a safe way to take no damage."
                   ]
+                },
+                {
+                  "name": "Back-Side Botwoon Microwave",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateUnderwater",
+                    "Charge",
+                    "Plasma",
+                    "canXrayWaitForIFrames"
+                  ],
+                  "reusableRoomwideNotable": "Back-Side Botwoon Magic Pixel Beam Fight",
+                  "note": [
+                    "Stand on the appropriate pixel for shooting diagonally through the wall and use the microwave trick to defeat Botwoon.",
+                    "Using angle up, it is where Samus' front foot is on the seam in the floor.",
+                    "There is not proper spacing for landing an angle down shot and xraying.",
+                    "Waiting for Botwoon to peak their head through the wall works too but is less safe."
+                  ]
+                },
+                {
+                  "name": "Back-Side Botwoon Magic Pixel",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateUnderwater",
+                    {
+                      "enemyKill": {
+                        "enemies": [ [ "Reverse Botwoon" ] ],
+                        "explicitWeapons": [ 
+                          "Charge+Plasma", 
+                          "Charge+Ice+Spazer"
+                       ]
+                      }
+                    }
+                  ],
+                  "reusableRoomwideNotable": "Back-Side Botwoon Magic Pixel Beam Fight",
+                  "note": [
+                    "A charge beam shot will pass right through the dividing wall if fired from the correct 2pixel window.",
+                    "Using angle down the spot is where Samus' front toe touches the wall.",
+                    "Using angle up, it is where Samus' front foot is on the seam in the floor."
+                  ]
+                },
+                {
+                  "name": "Back-Side Botwoon Plasma Shield Fight",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateUnderwater",
+                    {
+                      "enemyKill": {
+                        "enemies": [ [ "Reverse Botwoon" ] ],
+                        "explicitWeapons": [ "Plasma Shield" ]
+                      }
+                    }
+                  ],
+                  "note": "Stand near the dividing wall and unleash the Plasma Special Beam Attack!",
+                  "devNote": "6 power bombs expected."
```

Comment:
I think we need to add another node for Reverse Botwoon 2, which doesn't do damage, but still has 1500 health. It will take more PBs if using SBA, or more supers if doing that strat. Similar to nodes 3+5.

Reply:
I'll look at adding more Botwoon entities.  The dodge rate isn't the same with all weapons and I've been abusing that.